extensions/vscode/schema/mason.yaml.schema.json

```diff
@@ -49,7 +49,7 @@
             "url": {
               "type": "string",
               "description": "The git url for the associated brick (https://github.com/felangel/mason).",
-              "pattern": "^https?://",
+              "pattern": "^(https?|ssh)://",
```

Comment:
I just took a look at the pubspec schema and they allow any string which makes sense because other formats like:
`git@github.com:felangel/mason.git` will also not be considered valid. So I think we can just follow suite and allow any string here